### ISSUE TITLE
nodes with colon support

### ISSUE THREAD

**Issue by iharh** [aep/pug-rs#1]
pug-rs supports attributes with colon, bug does not support nodes with colon:
```
soapenv:Envelope(xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/")
    soapenv:Header
    soapenv:Body

```

**Comment by iharh** [comment]
Thank you very much.
Now pug-rs looks more like a tool, not like a library
BTW - do you have a plan to make parse-function usable (probably export it)?

**Comment by aep** [comment]
yeah, ideally this would be an npm package with a native extension. 
I'm not planning on doing that. Contributions are always welcome :)

**Comment by iharh** [comment]
Thank you very much. Maybe, one day I'll try to make a pure-rust lib for handling jade/pug templates...
